Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -340,13 +340,23 @@ - (BOOL)searchControllerShouldBeginSearch:(SPSearchController *)controller
     }
 
     bSearching = YES;
-    [self.tableView reloadData];
+
+    // TODO: Nuke the following as soon as we can!
```

Comment:
I'm not opposed to `TODO` comments but I recently removed one that said "remove this when it's no longer necessary" and I just had to guess that since it was about six years old that it wasn't necessary.

can we include the specific conditions in the `TODO` that communicate when it will be time to adjust the code?

```objective-c
// TODO: Nuke as soon as advanced search is deployed and CoreData tables are getting recent searches
```

Reply:
In this particular case, definitely no need to extend the comment further. Reason is... the second Avanced Search is ready, all of the `FeatureManager` references will be nuked (and these comments alongside!!). That is, this isn't an isolated patch, of part of a quite big set of changes.

That's hopefully next friday! Yay!!